components/board/genericlinux/gpio.go

```diff
@@ -58,6 +76,8 @@ func (pin *gpioPin) Get(ctx context.Context, extra map[string]interface{}) (resu
 	pin.mu.Lock()
 	defer pin.mu.Unlock()
 
+	pin.pwmRunning = false
```

Comment:
`Get` is for  using a GPIO pin as an input (you're reading the signal from the external world), and PWM stuff is for using it as an output (you're sending a signal out to the world). What would it mean to use a GPIO pin for both simultaneously?

...looking through the [pigpio source code](https://github.com/joan2937/pigpio/blob/c33738a320a3e28824af7807edafda440952c05d/pigpio.c#L8931), it looks like they only set the direction of a pin to PI_INPUT when using the pin as part of an I2C or SPI bus, and keep it as PI_OUTPUT for everything else (including in `gpioRead`). I guess it's not dangerous to keep the PWM loop running, though I'm unsure what the use case for this would be.